styles/style.css

```diff
@@ -0,0 +1,327 @@
+* {
+  box-sizing: border-box;
+  margin: auto;
+}
+main {
+  padding: 0.5rem 9.5rem 4rem 9.5rem;
+}
+img {
+  max-width: 100%;
+}
+ul {
+  list-style: none;
+}
+a:link {
+  text-decoration: none;
+}
+/* header style start */
+header > div {
+  background-image: url('../images/header-image.jpg');
+  display: grid;
+  padding-left: 5rem;
+  padding-right: 5rem;
+  padding-top: 2rem;
+  padding-bottom: 6rem;
+  grid-template-rows: 1fr 1fr 1fr 1fr;
+  grid-template-columns: 36% 1fr 25%;
+}
+header img[src="images/adoptadog-logo.svg"] {
+  max-width:100%;
+}
+header img[src="images/cart-icon.svg"] {
+  width: 35px;
+  padding-right: 0.4rem;
+}
+header .cart-icon-money {
+  display: flex;
+  flex-direction: row;
+}
+header p {
+  color: white;
+  font-size: 0.8rem;
+}
+header>div div:first-child {
+  grid-area: 1 / 1 / 5 / 2;
+}
+header div h1 {
+  color: #81B741;
+  font-size: 1.2rem;
+}
+header>div div:nth-child(2) {
+  grid-area: 1 / 3 / 2 / 4;
+}
+header>div button {
+  grid-area: 2 / 3 / 3 / 4;
+  background-color: #81B741;
+  border: none;
+  width: 5rem;
+  color: white;
+  width: 7rem;
+  height: 1.5rem;
+}
+/* header style end */
+/* nav style start */
+header nav {
+  background-color: #EBEBEB;
+  padding-left: 9.5rem;
+}
+header nav ul li {
+  display: inline-flex;
```

Comment:
just display: flex; is ok

Reply:
uhh, I changed it to flex, but the layout broken.